src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/template.json

```diff
@@ -2,21 +2,21 @@
   "$schema": "http://json.schemastore.org/template",
   "author": "Uno Platform",
   "classifications": [
-    "Web",
+    "Cross-platform",
     "Uno Platform",
-    "WebAssembly",
     "Android",
     "iOS",
     "Windows",
     "macOS",
     "Linux",
     "Tizen",
```

Comment:
There seems to be some inconsistencies for the classifications, where some have both `Linux` and `Tizen`, and some just `Linux`. Is this intended?

Reply:
Yes, some templates support Skia/GTK but not Skia/Tizen.